src/main/java/frc/team2412/robot/subsystem/TargetLocalizer.java

```diff
@@ -91,7 +91,8 @@ public double getAngularVelocity() {
      * @return adjustment
      */
     public double yawAdjustment() {
-        return Math.toDegrees(Math.asin(getLateralVelocity() / getDistance() * TURRET_LATERAL_FF))
+        return (getDistance() != 0 ? Math.toDegrees(Math.asin(getLateralVelocity() / getDistance() * TURRET_LATERAL_FF))
```

Comment:
Note that `Math.asin()` will return `NaN` for any values with an absolute value greater than one (in this case, when `getLateralVelocity() > getDistance()`), which may cause some unexpected behavior (even multiplying zero by `NaN` will still result in `NaN`, so setting `TURRET_LATERAL_FF` to zero won't prevent this issue).